ACMILabs/interactive-label
# #130 Debug mode for touchscreen testing

Resolves #130 

Allows for the visualising of touch screen areas/interactions.

### Acceptance Criteria
- [x] Able to see where the user last clicked
- [x] Able to see the clickable regions for the label

### Relevant design files
* None

### Testing instructions
1. Set `DEBUG=True` in your env
2. Load any Interactive Label
3. See that clickable regions are highlighted in green
4. See that a blue dot appears on the last clicked position on the screen
5. Restart the label with `DEBUG=False` and see everything behaves as normal
